Source/Examples/ExampleLibrary/Examples/PlotModelExamples.cs

```diff
@@ -67,6 +67,24 @@ public static PlotModel TitlesCenteredWithinView()
             return model;
         }
 
+        [Example("TitleClippingLength")]
+        public static PlotModel TitleClippingLength()
+        {
+            var model = new PlotModel { Title = "Title", TitleClippingLength = 0.6 };
```

Comment:
Could these have a really long title, so that it is more obvious what is going on?